uniffi_bindgen/src/bindings/ruby/templates/ObjectTemplate.rb

```diff
@@ -1,22 +1,46 @@
 class {{ obj.name()|class_name_rb }}
-  {%- match obj.primary_constructor() %}
-  {%- when Some with (cons) %}
-  def initialize({% call rb::arg_list_decl(cons) -%})
-    {%- call rb::coerce_args_extra_indent(cons) %}
-    handle = {% call rb::to_ffi_call(cons) %}
-    ObjectSpace.define_finalizer(self, self.class.define_finalizer_by_handle(handle))
-    @handle = handle
+
+  # A private helper for initializing instances of the class from a raw pointer,
+  # bypassing any initialization logic and ensuring they are GC'd properly.
```

Comment:
@saks I pulled out some "private" helpers here to make codegen a bit easier. Is there a pattern we should use to discourage consumers from calling this, similar to e.g. the leading-underscore pattern from Python?

Reply:
@rfk The best way to hide it would be to mark these methods as `private_class_method`, but this is not quite feasible in our case. Leading-underscore is also pretty common in ruby community, so employing it here is rather a good idea.

Reply:
Great, I'll by some leading underscores in here to more clearly communicate intent.